test/assert/assert.go

```diff
@@ -0,0 +1,36 @@
+// Copyright © 2020 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package assert
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+	"gotest.tools/v3/icmd"
+)
+
+// Cmd verifies result against expected(exit code)
+func Cmd(t *testing.T, result *icmd.Result, expected icmd.Expected) {
+	result.Assert(t, expected)
+}
+
+// NilError fails the test immediately if err is not nil.
+func NilError(t *testing.T, err error, msgAndArgs ...interface{}) {
+	assert.NilError(t, err, msgAndArgs)
+}
+
+func Assert(t *testing.T, comparison interface{}, msgAndArgs ...interface{}) {
+	assert.Assert(t, comparison, msgAndArgs...)
+}
```

Comment:
How much value do this add ? (aka the signature are almost the same, there is not extra "work", we could use what they do directly in the code :angel: )

Reply:
Agree @vdemeester I will remove this wrapper